Dockerfile

```diff
@@ -20,9 +48,7 @@ COPY build/*.whl /usr/local/sqlflow/python/
 RUN pip install /usr/local/sqlflow/python/*.whl
 
 # Install the pre-built binaries
-COPY build/sqlflowserver /usr/local/bin
-COPY build/sqlflow /usr/local/bin
-COPY build/step /usr/local/bin
+COPY build/sqlflowserver build/sqlflow build/step /usr/local/bin
```

Comment:
Why not adding `build/sqlflow` as the cli?

Reply:
I did copy build/sqlflow to /usr/loca/bin. Not sure if this answers your question.